pkgs/development/r-modules/default.nix

```diff
@@ -397,6 +397,7 @@ let
     RPushbullet = [ pkgs.which ];
     qtpaint = [ pkgs.cmake ];
     qtbase = [ pkgs.cmake pkgs.perl ];
+    RcppEigen = lib.optionals stdenv.isDarwin [ pkgs.libiconv ];
```

Comment:
Unless you are trying to avoid a mass rebuild, I would recommend getting rid of the conditional. libiconv is set up to  give you the right version.

Reply:
Okay, I see.